extensions/omni_seq/tests/test.yml

```diff
@@ -126,7 +126,7 @@ tests:
           omni_seq.id_int64_nextval(10, 'seq')
         )
     - query: |
-        insert into t select from generate_series(1, 10) returning id
+        insert into t select from generate_series(0, 10) returning id
```

Comment:
I noticed the original sql tests started the series from 0, I don't know if it's necessary to do that here so I can revert if it's not.

Reply:
I made this change to avoid oddly looking `11` there. I will revert this, but thanks for bringing attention to it! Details do matter.